tests/data/PS-or-PET-Raman-Spectrum-2.csv

```diff
@@ -1,1617 +0,0 @@
-//Exported ASCII-File

```

Comment:
Why are these deleted? If this is actual measurement data, it makes sense to keep it here for testing.